app/src/main/java/com/android/feedme/model/data/Ingredient.kt

```diff
@@ -1,3 +1,8 @@
 package com.android.feedme.model.data
 
-data class Ingredient(val name: String, val type: String, val id: String)
+data class Ingredient(
+    val name: String,
+    var id: String = "NO_ID",
```

Comment:
I think it could be better to encode the type of the ingredient as `val type: IngredientType = VEGAN` with `IngredientType` being an enum that we can extend. Otherwise future changes are gonna be really complicated :(

Reply:
I agree with you. However, I thing this should be in another PR because refactoring Ingredient demands a lot of energy and it isn't the main point of this PR.  I think we should also have the whole group agree on how to do it before refactoring. What do you think?

Reply:
Ok, is think that because we are gonna refactor the database tommorow, it's useless to do it now. Let's keep that for another PR.